.docksal/commands/release

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+## Make a new release and deploys it.
+##
+## Usage: fin release
+
+# Abort if anything fails
+set -e
+
+# Backup the environments.
+fin terminus backup:create ${hostingplatform}.dev --keep-for=365
+fin terminus backup:create ${hostingplatform}.test --keep-for=365
+fin terminus backup:create ${hostingplatform}.live --keep-for=365
+
+# Grab the live db.
+fin pull db -y --hosting-env=live
```

Comment:
Instead of the pulling the db what if you just use drush to export it to the private files directory and use rsync to pull it down. Could be potentially quicker.

Reply:
`terminus drush ${hostingsite}.live -- cex --destination=sites/default/files/private/export-2023-12-20 -y`
`terminus rsync ${hostingsite}.live:files/private/export-2023-12-20 ./`

Reply:
This was moved to the proposed `config-capture` command.